tests/SBRPTests/SBRPTests.csproj

```diff
@@ -0,0 +1,16 @@
+<Project Sdk="Microsoft.NET.Sdk">
+
+  <PropertyGroup>
+    <TargetFramework>$(NetCurrent)</TargetFramework>
+    <Nullable>enable</Nullable>
+    <IsPackable>false</IsPackable>
+    <IsUnitTestProject>true</IsUnitTestProject>
```

Comment:
Arcade already offers an in-built way to define test projects: https://github.com/dotnet/arcade/blob/234e0726c7384ee84bf08550f2d16a1ff2d5c543/src/Microsoft.DotNet.Arcade.Sdk/tools/Tests.props#L16

Please rename this project to something `.Tests.csproj` so that Arcade automatically recognizes it. Also remove the package references below, including the Versions.props entries for it. All of that is already handled by arcade.

Reply:
Got it. Thanks for letting me know this feature of Arcade!